### DISCUSSION TITLE
Annual Subscription Plan

### DISCUSSION THREAD

**Discussion by jeff282** [burkybang/SuiteAdvanced-Feedback#86] [Ideas]
Would love an option for an annual subscription plan if possible.

**Comment by burkybang** [comment]
I would love to implement that. In order to do that, I have two options:
1. Wait for my payment provider to support that (They say it's in the works, but no ETA)
2. Build my own payment system (This would cost a lot of time and money)

So far, I've been going with option 1. Hoping it will happen sooner than later 🤞

**Comment by burkybang** [comment]
I offer this now!

If you have an existing subscription, you'll need to cancel it. Then you can sign up for an annual subscription.
